docs/detections/rules-ui-create.asciidoc

```diff
@@ -263,33 +263,37 @@ For example, if a rule has an interval of 5 minutes, no additional look-back tim
 [[create-esql-rule]]
 === Create an ES|QL rule
 
-IMPORTANT: This is a placeholder for future documentation. The following content is incomplete.
+preview::[]
 
 . Go to *Rules* -> *Detection rules (SIEM)* -> *Create new rule*. The *Create new rule* page displays.
-. To create a rule that uses ES|QL, select **ES|QL**, 
-then write a query. There are two types of ES|QL queries:
-
-.. **Aggregating query**: This is a query that uses the `STATS...BY` grouping commands. Query results cannot be matched with a particular document in Elasticsearch. For example:
+. Select **ES|QL**. 
+. Write an aggregating or non-aggregating type query:
 +
-[esql]
------
+TIP: Click the **ES|QL reference** button [book icon] for more information on ES|QL syntax. 
++
+** **Aggregating query**: Uses {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate alert data. An alert is generated for each segment of the query, but is not associated with a specific source event since aggregating queries process data from multiple alert documents. As a result, alerts from aggregating queries do not contain source event data.
```

Comment:
> An alert is generated for each segment of the query,

What is meant by `each segment of the query`?

Reply:
I would try to rephrase it as: Generated alerts contain only fields returned by query, without mentioning source event documents. Let's leave it for non-aggregating query only,

Reply:
@vitaliidm to clarify, are you suggesting that the second and third sentences be revised like so:

```suggestion
** **Aggregating query**: Uses {ref}/esql-functions-operators.html#esql-agg-functions[`STATS...BY`] functions to aggregate alert data. An alert is generated for each row in the result table, and alert documents only contain fields returned by the query.  
```